src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileReaderService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String CORRECT_INPUT_FILE_NAME = "src/main/resources/fruits.csv";
+    private static final String WRONG_INPUT_FILE_NAME = "src/main/resources/fruits100.csv";
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void initialize() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void readingInformationFromNotExistingFile_notOk() {
```

Comment:
naming methodUnderTest_state_expectedBehavior
for example readRowsFromFile_NotExistingFile_notOk

Reply:
Agree